File: android/app/src/main/java/com/npkompleet/beatz/models/Album.java
package com.npkompleet.beatz.models;

public class Album {
    private long id;
    private String album;
    private String artist;
    private String albumArt="";
    private int numOfSongs;

    public Album(){}

    public Album(int id, String album, String artist, String albumArt, int numOfSongs){
        this.id = id;
        this.album = album;
        this.artist = artist;
        this.albumArt = albumArt;
        this.numOfSongs = numOfSongs;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getAlbum() {
        return album;
    }

    public void setAlbum(String album) {
        this.album = album;
    }

    public String getArtist() {
        return artist;
    }

    public void setArtist(String artist) {
        this.artist = artist;
    }

    public String getAlbumArt() {
        return albumArt;
    }

    public void setAlbumArt(String albumArt) {
        this.albumArt = albumArt;
    }

    public int getNumOfSongs() {
        return numOfSongs;
    }

    public void setNumOfSongs(int numOfSongs) {
        this.numOfSongs = numOfSongs;
    }
}

File: lib/models/album.dart
class Album {
  int id;
  String album;
  String artist;
  String albumArt;
  int numOfSongs;

  Album(this.id, this.album, this.artist, this.albumArt, this.numOfSongs);

  Album.fromJson(Map<String, dynamic> json) {
    album = json['album'];
    albumArt = json['albumArt'];
    artist = json['artist'] == "<unknown>" ? "Unknown" : json['artist'];
    id = json['id'];
    numOfSongs = json['numOfSongs'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['album'] = this.album;
    data['albumArt'] = this.albumArt;
    data['artist'] = this.artist;
    data['id'] = this.id;
    data['numOfSongs'] = this.numOfSongs;
    return data;
  }
}
